packages/website-eslint/src/mock/assert.js

```diff
@@ -0,0 +1,79 @@
+class AssertionError extends Error {
```

Comment:
is there a licence header we should embed in this file?

Reply:
hmm, i think we should just copy, same as i did in mock/path.js
https://github.com/nodejs/node/blob/master/LICENSE